policy-report/image-vuln-adapter/README.md

```diff
@@ -0,0 +1,55 @@
+# image vulnerability adapter
+The image vulnerability adapter runs an image vulnerability scanner tool called [trivy](https://github.com/aquasecurity/trivy) to scan a given kubernetes pod or workload to produces a namespace policy report based on the [Policy Report Custom Resource Definition](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report)
+
+## Installing
+Build the imgvuln binary: `make imgvuln`
+
+Type `imgvuln` command in your terminal to confirm if the image vulnerability adapter is installed already
+
+**Note**:
+* If it brings an error, the build is completed in the bin folder in the project, you just need to move the binary to your `/usr/local/bin` directory 
+* You can remove or delete from your `urs/local/bin` directory.
+
+## Running
+
+**Prerequisites**: 
+* Kubernetes Cluster: To run the Kubernetes cluster locally, tools like [kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/start/) can be used. Here are the steps to run the image vulnerability adapter with getting a `kind` cluster up and running.
+
+### Steps
+
+#### Common steps
+```sh
+# 1. Create a Kubernetes cluster
+kind create cluster --name mycluster --image <kindest/node:stpecify version tag>
+
+# 2. Create a policy report CustomResourceDefinition
+kubectl create -f kubernetes/crd/v1alpha2/wgpolicyk8s.io_policyreports.yaml
+
+# 3. Create a vulnerability report CustomResourceDefinition
+imgvuln init
```

Comment:
Can we install the policy report CRD as part of the `init` command?

In fact it would be best to not require the user to run the `init` command but simply check for the required resources are create them if not present.

Reply:
ok... no more init command.